### ISSUE TITLE
Plugin breaks when used with Gradle 7.0

### ISSUE THREAD

**Issue by matejdro** [bjoernQ/unmock-plugin#70]
When attempting to use unmock with Gradle 7.0, build fails with the following error:

```
 Some problems were found with the configuration of task ':location:unMock' (type 'UnMockTask').
   - Type 'UnMockTask' property 'outputDir' is annotated with @PathSensitive but that is not allowed for 'OutputDirectory' properties.
     
    Reason: This modifier is used in conjunction with a property of type 'OutputDirectory' but this doesn't have semantics.
    
    Possible solution: Remove the '@PathSensitive' annotation.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#incompatible_annotations for more details about this problem.

  - Type 'UnMockTask' property 'unmockedOutputJar' is annotated with @PathSensitive but that is not allowed for 'OutputFile' properties.
    
    Reason: This modifier is used in conjunction with a property of type 'OutputFile' but this doesn't have semantics.
    
    Possible solution: Remove the '@PathSensitive' annotation.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#incompatible_annotations for more details about this problem.
```

**Comment by bjoernQ** [comment]
Will look into this but I'm a quite busy currently - if anyone likes to look into this I'm happy to receive a PR for this

**Comment by calvarez-ov** [comment]
> Will look into this but I'm a quite busy currently - if anyone likes to look into this I'm happy to receive a PR for this

Here's one, hope it helps: https://github.com/bjoernQ/unmock-plugin/pull/71

**Comment by bjoernQ** [comment]
I merged #71 and published it as version 0.7.8 - apparently the sync to Maven Central takes some time - hope the sync goes through soon

thanks for reporting and thanks for the PR

**Comment by bjoernQ** [comment]
sync is done: https://repo1.maven.org/maven2/com/github/bjoernq/unmockplugin/0.7.8/

will close this issue now
